wordpress-mobile/WordPress-iOS
# Lock Screen Widgets: Remove background and update UI

Part of https://github.com/wordpress-mobile/WordPress-iOS/issues/21672

Making changes based on iOS 17 preference to present lock screen widgets without background. 

iOS 17 API conformance will be done in another PR when we enable Xcode 15 CI builds. 

Discussion: pc8eDl-14z-p2#comment-937

### Updates

- Removed AccessoryWidgetBackground
- Increased font sizes
- Adjusted spacing between elements
- Added Jetpack Icon

| 1  | 2  |
|---|---|
| ![IMG_1065](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/a2404a85-41bf-41e3-bd36-2b6b2330c03d)  | ![IMG_1063 2](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/e78d791a-cc3c-4559-b29e-580e1dbcc968)  |

## To test:

Most comfortable to test with AppCenter builds or locally on Xcode 14. Add Lock Screen Widgets, change sites, confirm all the data is visible and it looks presentable.

## Regression Notes
1. Potential unintended areas of impact

Previously developed lock screen widget functionality should continue working

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing

3. What automated tests I added (or what prevented me from doing so)

None

PR submission checklist:


UI Changes testing checklist:
